template.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<head>
+<title>
+__TITLE__
+</title>
+<link rel='stylesheet' href='https://cdn.jsdelivr.net/gh/zyguan/railroad-diagrams@5e868a958ba52847e28b2adefc5dcd8b3172c58d/railroad.css'/>
+<style>
+  body {
+    background-color: hsl(30, 20%, 95%);
+  }
+  svg.railroad-diagram rect {
+    fill: rgb(245, 245, 245);
+  }
+  svg.railroad-diagram g.color-green rect {
+    fill: rgb(204, 255, 204);
+  }
+</style>
+<script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+<script src="https://cdn.jsdelivr.net/gh/zyguan/railroad-diagrams@5e868a958ba52847e28b2adefc5dcd8b3172c58d/railroad.umd.js"></script>
+</head>
+<body>
+
+<div id="railroad-tooltip" class="railroad-diagram-tooltip"></div>
+<div id="CONTENT"></div>
+<script id="MARKDOWN" type="text/plain">
+__MARKDOWN__
+</script>
+
+<script>
+  // export railroad defaults
+  Object.entries(railroad.default).forEach(([k, v]) => globalThis[k] = v);
+  // aliases
+  function Span(text, {href, tooltip, color}={}) {
```

Comment:
Need to replace all `NonTerminal` with `Span` in the future?

Reply:
I've already replaced them. Here I mean to define shortcuts for some common usages, I think we can still use the builtin diagram node (like `NonTerminal`) if needed.